### PR TITLE
[zh] Translate common-definitions/list-meta page into Chinese

### PR DESCRIPTION
**Pull request by twilight0620**
Fixes: https://github.com/kubernetes/website/issues/32247

Do not have chinese docs: https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/list-meta/

so need to translate common-definitions/list-meta page into Chinese

### CODE REVIEW

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+api_metadata:
+  apiVersion: ""
+  import: "k8s.io/apimachinery/pkg/apis/meta/v1"
+  kind: "ListMeta"
+content_type: "api_reference"
+description: "ListMeta 描述了合成资源必须具有的元数据，包括列表和各种状态对象。"
+title: "列表元"
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:8]:
```suggestion
title: "ListMeta"
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+<!--
+- **continue** (string)
+
+  continue may be set if the user set a limit on the number of items returned, and indicates that the server has more data available. The value is opaque and may be used to issue another request to the endpoint that served this list to retrieve the next set of available objects. Continuing a consistent list may not be possible if the server configuration has changed or more than a few minutes have passed. The resourceVersion field returned when using this continue value will be identical to the value in the first response, unless you have received this token from an error message.
+-->
+
+- **continue** (string)
+
+  如果用户对返回的项目数量设置了限制，则可以设置 `continue`，表示服务器有更多可用的数据。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:44]:
```suggestion
  如果用户对返回的条目数量设置了限制，则 `continue` 可能被设置，表示服务器有更多可用的数据。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  该值是不透明的，可用于向服务于此列表的端点发出另一个请求，以检索下一组可用的对象。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:45]:
```suggestion
  该值是不透明的，可用于向提供此列表服务的端点发出另一个请求，以检索下一组可用的对象。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  如果服务器配置已更改或已过去几分钟，则可能无法继续保持一致的列表。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:46]:
```suggestion
  如果服务器配置已更改或时间已过去几分钟，则可能无法继续提供一致的列表。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  除非您从错误消息中收到此令牌（token），否则使用此 `continue` 值时返回的 `resourceVersion` 字段应该和第一个响应中的值是相同的。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:47]:
```suggestion
  除非你在错误消息中收到此令牌（token），否则使用此 `continue` 值时返回的 `resourceVersion`
  字段应该和第一个响应中的值是相同的。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+<!--
+- **resourceVersion** (string)
+
+  String that identifies the server's internal version of this object that can be used by clients to determine when objects have changed. Value must be treated as opaque by clients and passed unmodified back to the server. Populated by the system. Read-only. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency
+-->
+
+- **resourceVersion** (string)
+
+  标识该对象的服务器内部版本的字符串，客户端可以用该字段来确定对象何时更改。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:71]:
```suggestion
  标识该对象的服务器内部版本的字符串，客户端可以用该字段来确定对象何时被更改。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+  该值对客户端是不透明的，并且未经修改地传回给服务器。该值由系统填充，只读。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:72]:
```suggestion
  该值对客户端是不透明的，并且应该原样传回给服务器。该值由系统填充，只读。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+  更多信息：https ://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency 。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:73]:
```suggestion
  更多信息：https ://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md (hunk 1 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:8]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md (hunk 2 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:44]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md (hunk 3 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:45]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md (hunk 4 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:46]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md (hunk 6 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:71]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md (hunk 5 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:47]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md (hunk 7 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:72]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md (hunk 8 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:73]: 已修改

**Comment by twilight0620** [comment]
I have modified all comments. Thanks for review.

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+- **resourceVersion** (string)
+
+  String that identifies the server's internal version of this object that can be used by clients to determine when objects have changed. Value must be treated as opaque by clients and passed unmodified back to the server. Populated by the system. Read-only. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency
+-->
+
+- **resourceVersion** (string)
+
+  标识该对象的服务器内部版本的字符串，客户端可以用该字段来确定对象何时被更改。
+  该值对客户端是不透明的，并且应该原样传回给服务器。该值由系统填充，只读。
+  更多信息：https ://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency。
```
Comment (ydFu) [inline content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md:74]:
```suggestion
  更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency。
```
URL link is not working

**Review by ydFu — COMMENTED** [review]
Please use rebase to squash commits into only one commit.

File: content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md (hunk 9 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/list-meta.md]: 已修改

**Comment by twilight0620** [comment]
> Please use rebase to squash commits into only one commit.

I have rebased it. Thanks for your review.

**Comment by ydFu** [comment]
/lgtm
Thank for your contribution.

**Pull request merged**